libs/settings.py

```diff
@@ -32,6 +35,7 @@
 # Default: half of available cores.
 N_JOBS = None
 
-# Amount of cores to use for low-computational tasks (IO, etc).
-# Default: all available cores.
+# Amount of cores to use for low-computational tasks (IO, etc). This number
+# could be higher than N_JOBS.
+# Default: same as N_JOBS.
 N_JOBS_HIGH = None
```

Comment:
I'm a little confused that it's called N_JOBS_HIGH if it's for low-computational tasks

Reply:
Yeah, you are right. I think it's late to change it now (since I have to change other files), but since I will use the base of this code for another project, I will rename this variable there!